components/fms-server/src/models.rs

```diff
@@ -515,4 +515,765 @@ pub struct ErrorObject {
 
 }
 
+#[derive(Debug, Clone, PartialEq, serde::Serialize, serde::Deserialize)]
```

Comment:
this module seems to be quite large. Can we distribute the type to multiple modules?

Reply:
Yes, we could distribute it. However, I was not sure where to split because this is basically the domain model used within rFMS. I will try to split this along the lines of the different endpoints using parts of the model.

Reply:
Yes, I guess that makes sense. Putting common types into one module and then have the endpoint specific types refer to these ...